lib/msal-core/src/authority/Authority.ts

```diff
@@ -87,6 +106,9 @@ export class Authority {
 
     // http://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata
     protected get DefaultOpenIdConfigurationEndpoint(): string {
+        if (this.AuthorityType === AuthorityType.Adfs){
+            return `${this.CanonicalAuthority}.well-known/openid-configuration`;
```

Comment:
Should we make ".well-known/openid-configuration" a constant?

Reply:
Done